Tests/scripts/sdk_pylint_check.sh

```diff
@@ -5,9 +5,9 @@ echo Checking for sdk related pylint errors
 pylint_tests_result="$(python3 -m pylint --errors-only ./Tests | grep demisto_sdk)"
 pylint_utils_result="$(python3 -m pylint --errors-only ./Utils | grep demisto_sdk)"
 
-if [ -n "$pylint_test_result" ] || [ -n "$pylint_utils_result" ]; then
-    echo Pylint result on Test directory: $pylint_test_result
-    echo Pylint result on Utils directory: $pylint_utils_result
+if [ -n "$pylint_tests_result" ] || [ -n "$pylint_utils_result" ]; then
```

Comment:
Why not split it to 2 different `if`s?
What if there are errors only in one of them, why print the other?

Reply:
Actually the other should be empty , ":"
But I will  did it in the dummy way.